homeassistant/helpers/script.py

```diff
@@ -235,6 +235,11 @@ def _log(
             msg, *args, level=level, **kwargs
         )
 
+    def _step_log(self, default_message, delay=None):
```

Comment:
Shouldn't `delay` be called `timeout`, as it's how long we're going to wait for something ?